### PR TITLE
Bundler chores

### PR DESCRIPTION
**Pull request by sschuberth**
Please have a look at the individual commit messages for the details.

### CODE REVIEW

File: plugins/package-managers/bundler/src/test/kotlin/BundlerTest.kt [hunk 1]
```diff
@@ -56,11 +56,11 @@ class BundlerTest : WordSpec({
         }
     }
 
-    "createFromJson()" should {
-        "parse JSON metadata for a Gem correctly" {
-            val rspecGemJson = File("src/test/assets/rspec-3.7.0.yaml")
+    "createFromGem()" should {
+        "parse YAML metadata for a Gem correctly" {
+            val rspecGemYaml = File("src/test/assets/rspec-3.7.0.yaml")
```
Comment (fviernau) [inline plugins/package-managers/bundler/src/test/kotlin/BundlerTest.kt:61]: Would it make sense to rename this to `gemSpecFile` while at it?

File: plugins/package-managers/bundler/src/main/kotlin/Bundler.kt [hunk 2]
```diff
@@ -453,13 +453,17 @@ internal data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val description = sequenceOf(node["description"], node["info"]).mapNotNull { text ->
+                text?.textValue()?.takeIf { it.isNotEmpty() }
+            }.firstOrNull()
```
Comment (fviernau) [inline plugins/package-managers/bundler/src/main/kotlin/Bundler.kt:458]: Would it make sense to have a code comment explaining this fallback?

Furthermore, would it make sense to adjust the fallback to: description -> summary -> info ?

File: plugins/package-managers/bundler/src/main/kotlin/Bundler.kt [hunk 3]
```diff
@@ -453,13 +453,17 @@ internal data class GemSpec(
                 RemoteArtifact.EMPTY
             }
 
+            val description = sequenceOf(node["description"], node["info"]).mapNotNull { text ->
+                text?.textValue()?.takeIf { it.isNotEmpty() }
```
Comment (fviernau) [inline plugins/package-managers/bundler/src/main/kotlin/Bundler.kt:457]: Should this use `textValueOrEmpty()` ?

**Review by fviernau — CHANGES_REQUESTED** [review]

File: plugins/package-managers/bundler/src/main/kotlin/Bundler.kt (hunk 3 above)
Comment (sschuberth) [reply plugins/package-managers/bundler/src/main/kotlin/Bundler.kt:457]: Both would work (as "" is turned to `null` via `takeIf { it.isNotEmpty() }`), but `textValueOrEmpty()` does more than necessary here, so I'd prefer to stick to just `textValue()`.

**Review by fviernau — APPROVED** [review]

**Pull request merged**